reference/pickup.v3.yaml

```diff
@@ -0,0 +1,99 @@
+openapi: 3.0.1
+info:
+  title: Pickup
+  version: '3.0'
+  termsOfService: 'https://www.bigcommerce.com/terms'
+servers:
+  - url: 'https://api.bigcommerce.com'
+    variables:
+      store_hash:
+        default: store_hash
+paths:
+  '/stores/{store_hash}/v3/pickup/methods':
+    delete:
+      tags:
+        - Pickup Methods
+      description: Delete existing pickup methods.
+      parameters:
+        - name: id:in
+          description: Comma separated list of pickup method IDs.
+          in: query
+          required: false
+          schema:
+            type: integer
+            format: int64
+        - name: store_hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Request has been successfully processed.
+      summary: Delete Pickup Methods
+    put:
+      tags:
+        - Pickup Methods
+      description: Update existing pickup methods.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/PickupMethod'
+      parameters:
+        - name: store_hash
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: The request has been successfully processed.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/PickupMethod'
+                  meta:
+                    type: object
+        '422':
+          description: The request contains invalid data and may only partially be fulfilled. More information can be found in the error description.
+components:
+  schemas:
+    PickupMethod:
```

Comment:
List which properties are required

Reply:
I was thinking that this can be kept as the base model as this can be used for responses (which don't have a required property). I've updated the api-spec and have included the base model specifically in the request here https://github.com/bigcommerce/api-specs/pull/607/files#diff-941a81c827338c2ca7d86849e24d30cb8c5057711d9ae9f7e0032050d21e9a9cR31-R35